File: biometric-service/server_test.js
// Node.js health check for biometric service
const fetch = require('node-fetch');
const BASE = 'http://localhost:8000';

(async function(){
  try{
    const res = await fetch(`${BASE}/api/health`);
    const body = await res.text();
    console.log('status', res.status);
    console.log(body);
  }catch(e){
    console.error('error', e);
  }
})();

File: biometric-service/server_test.py
# Simple script to test biometric service endpoints (python)
# Use `python server_test.py` after starting the biometric service to perform a basic health check.
import requests

BASE = "http://localhost:8000"

def health():
    r = requests.get(f"{BASE}/api/health")
    print('health', r.status_code, r.text)

if __name__ == '__main__':
    health()
